README.md

```diff
@@ -31,7 +31,7 @@ variable exists and is set to a value other than 0.
 
 # Example #
 
-There are two examples in the `examples` directory but in short:
+There are three examples in the `examples` directory but in short:
```

Comment:
Please could you correct the example below. e.g. There's a comment that says `add_kernel` expects pre-compiled kernels but that's no longer true.

Reply:
Done